### ISSUE TITLE
Troubles login

### ISSUE THREAD

**Issue by mishoboss** [cubieplayer/Cubian#106]
When I try to login on Cubian booted on SD card via SSH, I get this:

login as: cubie
We'd like to keep you up to date about:
- Software feature updates
- New product announcements
- Special events
  Please register your products now at:  www.hp.com/networking/register

HP J8698A Switch E5412zl
Software revision K.15.08.0008

Copyright (C) 1991-2012 Hewlett-Packard Development Company, L.P.

```
               RESTRICTED RIGHTS LEGEND
```

 Confidential computer software.  Valid license from HP required for possession,
 use or copying. Consistent with FAR 12.211 and 12.212, Commercial Computer
 Software, Computer Software Documentation, and Technical Data for Commercial
 Items are licensed to the U.S. Government under vendor's standard commercial
 license.
                   HEWLETT-PACKARD DEVELOPMENT COMPANY, L.P.
                   20555 State Highway 249, Houston, TX 77070
## Press any key to continue

I'm pretty sure I SSH the right IP address.

**Comment by ghost** [comment]
It appears you are signing into the printer network. If you are able to get into the router and check DHCP, you will see the correct IP of your cubie. Also check arp -a to show your network connections.

**Comment by mishoboss** [comment]
This is exactly what I did... and the strange thing is I don't even have a printer network.

**Comment by cubieplayer** [comment]
This is really wired

**Comment by 346L3** [comment]
Do you have a HP ProCurve (Smart Network Switch) attached anywhere in the Network? If so, a likely explanation would be, that it has a static IP set for its management Interface, but blocks Pings. Thus, no DHCP-lease exists for the switch and when the DHCP-server tries the IP before it assigns it to the Cubieboard, it "seems" to be free, because no reply to the Ping request occurs... When you actually SSH to the IP, the switch will respond...

**Comment by mishoboss** [comment]
Nope... I have a stupid low-cost TP-Link router. That's all. Nothing HP manufactured except one of the laptops. This evening I'll try to assign a static IP to the cubie. Thanks.

**Comment by mishoboss** [comment]
I made the router assign a specific IP address to cubieboard and I logged in successfully now. It's very strange! I don't have a printer or anything HP related in my network. However I highly doubt it's a cubian issue, so you may close or delete this issue. Than you for the effort!
